src/docs/reference/modules/Notifications/README.md

```diff
@@ -0,0 +1,32 @@
+# Notifications (`OrchardCore.Notifications`)
+
+The `Notifications` module provides the infrastructure necessary to send notification to the app users.
+
+## Notification Methods
+
+There are many methods to send notifications to a user (e.x., Web, Email, Push, SMS, etc.). In addition to web notification, OrchardCore is shipped with Email based notifications. To allow users to receive Email notification, enable the `Email Notifications` feature. Note, you must also configure the [SMTP Settings](../Email/README.md). When multiple notification methods are enabled, the user can opt-in or opt-out any method he/she wish to receive by editing their profile.
```

Comment:
wish => wishes

Reply:
You can actually add a Note if you want.
Ex:
https://github.com/OrchardCMS/OrchardCore/blob/86ee333f0ac939fe85128782d2a59c88d435d90e/src/docs/getting-started/README.md?plain=1#L9

Reply:
@agriffard I updated the docs